lit/project/architecture.lit

```diff
@@ -1,78 +0,0 @@
-\title{Architecture}{architecture}
```

Comment:
It seems like the removal of this file is preventing the rendering of `/`:

```
failed to load section: stat lit/project/architecture.lit: no such file or directory
```

I guess that's because it's referenced in few other places 🤔  

```
ag -Q "architecture"                                                                        [0/54]
lit/docs/internals.lit
13:\inline-header{Basic architecture}
20:\tutorial-image{images/concourse_architecture.png}
71:  registering \reference{architecture-worker}{workers} with the
92:  \title{Workers\aux{: container runtime & cache management}}{architecture-worker}

lit/project/contribute.lit
47:  getting started guide. You can also get an overview of the Concourse architecture
48:  by reviewing some of the documentation under \link{Concourse Architecture}{architecture.html}.

lit/project.lit
79:\include-section{./project/architecture.lit}
```